libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -511,18 +529,44 @@ class MapboxNavigation(
          * Use [NavigationOptions.toBuilder] to easily customize selected options.
          */
         @JvmStatic
-        fun defaultNavigationOptions(context: Context): NavigationOptions {
-            return NavigationOptions.Builder(
-                NONE_SPECIFIED,
-                ROUNDING_INCREMENT_FIFTY,
-                DEFAULT_NAVIGATOR_POLLING_DELAY,
-                MapboxDistanceFormatter(
-                    context.applicationContext,
-                    null,
-                    UNDEFINED,
-                    ROUNDING_INCREMENT_FIFTY
+        fun defaultNavigationOptions(context: Context, accessToken: String?): NavigationOptions {
+            val tilesUri = URI("https://api-routing-tiles-staging.tilestream.net")
+            val tilesVersion = "2019_04_13-00_00_11"
+            return NavigationOptions.Builder()
```

Comment:
could we strip this out of MapboxNavigation? what we want from the external app is to pass this same configuration, with different `tilesUri` and `tilesVersion`. but everything else is the same

Reply:
`defaultNavigationOptions` is a public method.

You can call `MapboxNavigation.defaultNavigationOptions(context, token).toBuilder()` and modify what's necessary.

Reply:
can we update this function definition in this PR then? because that's all we need

```
val defaultTilesHost = "https://api-routing-tiles-staging.tilestream.net"
val defaultTilesVersion = "2019_04_13-00_00_11"
fun defaultNavigationOptions(context: Context, tilesHost: String, tilesVersion: String, accessToken: String?):
```

Reply:
another option is to return the `NavigationOptions.Builder()`. and then `MapboxNavigation` will `.build()` it